source/isDefined.js

```diff
@@ -0,0 +1,23 @@
+import isNil from './isNil';
+import complement from './complement';
+
+
+/**
+ * Checks if the input value is not `null` and not `undefined`.
+ *
+ * @func
+ * @memberOf R
+ * @since v0.26.
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} x The value to test.
+ * @return {Boolean} `true` if `x` is not `undefined` or not `null`, otherwise `false`.
+ * @example
+ *
+ *      R.isDefined(null); //=> false
+ *      R.isDefined(undefined); //=> false
+ *      R.isDefined(0); //=> true
+ *      R.isDefined([]); //=> true
+ */
+var isDefined = complement(isNil);
```

Comment:
I see no reason to depend on `complement` here.   Just do it manually.

Reply:
Removed in https://github.com/ramda/ramda/pull/2818/commits/0789ed369b2e0c256a76adeda38bbdc31d956919